learningresources/models.py

```diff
@@ -88,5 +92,20 @@ def save(self, *args, **kwargs):
                 slug = "{0}{1}".format(slugify(self.name), count)
                 count += 1
             self.slug = slug
+            is_create = True
+        new_repository = super(Repository, self).save(*args, **kwargs)
+        if is_create:
+            roles_init_new_repo(self)
+        return new_repository
 
-        return super(Repository, self).save(*args, **kwargs)
+    class Meta:
+        # pylint: disable=missing-docstring
+        # pylint: disable=old-style-class
```

Comment:
old-style-class, no-init, and too-few-public-methods are now disabled globally so you can remove these lines